tests/cli/test_agent.py

```diff
@@ -1,17 +1,45 @@
+import warnings
 from unittest.mock import ANY
 
+import pytest
+
 import prefect.cli.agent
 from prefect import PrefectClient
+from prefect._internal.compatibility.deprecated import PrefectDeprecationWarning
 from prefect.settings import PREFECT_AGENT_PREFETCH_SECONDS, temporary_settings
 from prefect.testing.cli import invoke_and_assert
 from prefect.testing.utilities import MagicMock
 from prefect.utilities.asyncutils import run_sync_in_worker_thread
 
 
+@pytest.fixture(autouse=True)
+def ignore_agent_deprecation_warnings():
+    """
+    Ignore deprecation warnings from the agent module to avoid
+    test failures.
+    """
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=PrefectDeprecationWarning)
+        yield
```

Comment:
I noticed we have this fixture in 4 places. Is there a reason it makes more sense to add this fixture to each file rather than `conftest.py`?

Reply:
Yup, that makes a lot of sense. I moved this fixture to `/test/fixtures`, so all tests will use it by default now.